crossword-xml-uploader/src/main/scala/com/gu/crossword/crosswords/XmlProcessor.scala

```diff
@@ -44,7 +46,7 @@ object XmlProcessor extends DateLogic {
       </externalReferences>
     </crossword>
 
-  }
+  }.toEither
```

Comment:
There are a couple of `Try`s added in this commit. From what I can tell in the `Left` case the `XML.loadString` method that's called in the `CrosswordUploader.scala` will return a message from the `XML` lib. But do we know what this `process` function will return for its `Left`? Should we add a message to make debugging easier?

Reply:
Converting from a `Try` to an `Either` always expects the left case to be a throwable (in this case some kind of XML parsing error). Subsequently i've moved the business of converting to `Either` up to the top, printing errors and their stack traces when summarising results.

Reply:
thanks! yep I just got to that commit -- I should have waited until I got to the end to post this!